pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[build-system]
+requires = ["setuptools>=43.0.0", "setuptools_scm[toml]>=6.2", "wheel",
+            "oldest-supported-numpy"]
```

Comment:
Unless this package has C code that needs `numpy`, then it won't be a build requirement, only a runtime requirement.  So you can get rid of `oldest-supported-numpy` here.